backend/config/settings_gcp.py

```diff
@@ -0,0 +1,10 @@
+from google.cloud import datastore
+
+client = datastore.Client()
+
+def get_setting(name):
```

Comment:
can we add comments specifying that the datastore is defined in GCP and the settings that this queries are reading are pre-defined externally (e.g. manually via the GCP web console)?